protos/yoda.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
```

Comment:
Causes top-level messages, enums, and services to be defined at the package level, rather than inside an outer class named after the .proto file.  This will prevent generation of one giant file, specially when the proto file is extensive.

Reply:
👍